### DISCUSSION TITLE
Streamable Head in +onRenderHtml

### DISCUSSION THREAD

**Discussion by phiberber** [vikejs/vike#873] [Discussion] [answered]
Seems `escapeInject` only allow one stream at a time, I'm implementing in the V1 `+config` a meta called title which is of type `string | (pageContext: PageContext) => String | Promise<string>` and I would like to stream the page title, which is part of the head of the page.

I don't think creating a ReadableStream that contains both head and body would be the answer.

**Comment by brillout** [comment]
I'm not sure there is a benefit to stream the head? Correct me if I'm wrong but I think that `<title>` needs to be awaited before starting the HTML stream of the body (otherwise browsers won't display it).

**Reply by phiberber** [reply]
I was hoping it doesn't. Sometimes the title of the page is going to depend of some data in the database, awaiting the title means delaying the page while the title isn't ready, which is the purpose of streaming, not needing to delay the whole page for something to load.

**Reply by brillout** [reply, marked as answer]
AFAIK there is no way to stream `<head>` content after starting streaming `<body>` content.

So I don't think there is way to achieve what you want.
